### ISSUE TITLE
Remove error banner for non-app symbols

### ISSUE THREAD

**Issue by kerenkhatiwada** [getsentry/sentry#46955]
### Problem Statement

Currently, Sentry is displaying an error banner for non-application symbols. This is causing frustration for some users who are unable to control or supply these symbols to Sentry. While adding built-in repositories can resolve some symbol processing issues, it does not remove the error banner. The request is for the ability to remove the error banner for non-application symbols. This would improve the user experience and reduce confusion for users who are unable to control or supply these symbols.

<img width="907" alt="image" src="https://user-images.githubusercontent.com/99350819/230160356-155cceb6-0dcd-48a8-90e3-203ada63be08.png">

**Comment by philipphofmann** [comment]
I fully agree, @kerenkhatiwada. I need to find the right team to take care of this.

**Comment by matejminar** [comment]
@ashwoods Do you need any help from us or is your team on top of this one?

**Comment by Swatinem** [comment]
These errors are collected directly after symbolicator here:
https://github.com/getsentry/sentry/blob/13759ac20b710e4fb34aee4665464e0d30ad90f7/src/sentry/lang/native/processing.py#L94-L118

That happen *before* grouping, so we have no info about in-app state of a frame or a module at all at that point.

**Comment by hubertdeng123** [comment]
Hey all, does this issue need to be rerouted 🤔?

**Comment by billyvg** [comment]
Similar ticket to https://github.com/getsentry/sentry/issues/47323
